src/components/AddNewUserModal.module.css

```diff
@@ -0,0 +1,33 @@
+@import '../common/global.css';
+
+/* .box {
+  width: 80vw;
+} */
+
+.profilePic {
+  background-color: white;
+  width: 4vw;
+  margin: auto;
+  margin-bottom: 3vh;
+  margin-top: 3vh;
+}
+
+.modalHeader {
+  margin: auto;
+}
+
+.buttonContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2vw;
+}
+
+.sendEmailButton {
+  background-color: var(--cds-blue-1);
+  color: white;
+}
+
+.buttonContainer button {
```

Comment:
change whatever buttons are being selected by this to a new class and then select the class itself here